app/api/order/models.py

```diff
@@ -5,8 +5,8 @@
 
 
 class OrderUpdate(SQLModel):
-    end_date: datetime = Field(default_factory=lambda: datetime.now(timezone.utc) + timedelta(days=90))
-    is_returned: bool = Field(default=False)
+    end_date: datetime | None = Field(default_factory=lambda: datetime.now(timezone.utc) + timedelta(days=90))
+    is_returned: bool | None= Field(default=False)
```

Comment:
For OrderUpdate we don't need this default values. They were used for creation, but on update you should only put like -> `is_returned: bool | None = None` and the same for `end_date`

Reply:
fixed